provider/data/decimal/fingerprints.csv

```diff
@@ -1,58 +1,38 @@
-decimal/symbols@1, <lookup>, 1984B, 252 identifiers
+decimal/symbols@1, <lookup>, 1141B, 214 identifiers
 decimal/symbols@1, <total>, 8905B, 45 unique payloads
 decimal/symbols@1, af, 197B, 8f773f51e85a65c1
 decimal/symbols@1, ar, 202B, c3f15eb63fa35608
 decimal/symbols@1, ar-AE, 202B, 29e2dc764329c56
-decimal/symbols@1, ar-AE-u-nu-arab, 202B, -> ar
-decimal/symbols@1, ar-BH-u-nu-latn, 202B, -> ar-AE
-decimal/symbols@1, ar-DJ-u-nu-latn, 202B, -> ar-AE
+decimal/symbols@1, ar-AE/arab, 202B, -> ar
 decimal/symbols@1, ar-DZ, 202B, d2f2336636ab14d7
-decimal/symbols@1, ar-DZ-u-nu-arab, 202B, -> ar
-decimal/symbols@1, ar-EG-u-nu-latn, 202B, -> ar-AE
+decimal/symbols@1, ar-DZ/arab, 202B, -> ar
 decimal/symbols@1, ar-EH, 202B, -> ar-AE
-decimal/symbols@1, ar-EH-u-nu-arab, 202B, -> ar
-decimal/symbols@1, ar-ER-u-nu-latn, 202B, -> ar-AE
-decimal/symbols@1, ar-IL-u-nu-latn, 202B, -> ar-AE
-decimal/symbols@1, ar-IQ-u-nu-latn, 202B, -> ar-AE
-decimal/symbols@1, ar-JO-u-nu-latn, 202B, -> ar-AE
```

Comment:
it looks like these didn't get deduplicated properly before

Reply:
I think it's because the fallback order was different.